frontend/javascripts/oxalis/model_initialization.ts

```diff
@@ -479,7 +497,7 @@ function setupLayerForVolumeTracing(
   // layers, we cannot render both at the same time. Hiding the existing segmentation
   // layer would be good, but this information is stored per dataset and not per annotation
   // currently. Also, see https://github.com/scalableminds/webknossos/issues/5695
-  const newLayers = originalLayers.filter((layer) => layer.category !== "segmentation");
+  const newLayers = originalLayers.slice(); // .filter((layer) => layer.category !== "segmentation");
```

Comment:
The comments behind and above this line (also the one in line 490) can be removed now, I think.